search-api/src/search_api/models/document.py

```diff
@@ -0,0 +1,66 @@
+# Copyright © 2022 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Table for storing document details."""
+from enum import Enum
+
+from .db import db
+
+
+class Document(db.Model):
+    """Used to hold the documents requested in an access request."""
+
+    class DocumentTypes(str, Enum):
+        """Enum of the document types."""
+
+        BUSINESS_SUMMARY_FILING_HISTORY = 'summary_history'
+        CERTIFICATE_OF_GOOD_STANDING = 'cogs'
+        CERTIFICATE_OF_STATUS = 'cert_of_status'
+        LETTER_UNDER_SEAL = 'letter_under_seal'
+
+    __versioned__ = {}
+    __tablename__ = 'document'
+
+    id = db.Column(db.Integer, primary_key=True)
+    document_type = db.Column(db.String(30), index=True)
+    document_key = db.Column('document_key', db.String(100), nullable=False)
+    file_name = db.Column('file_name', db.String(100))
+    file_key = db.Column('file_key', db.String(100))
+    access_request_id = db.Column('access_request_id', db.Integer, db.ForeignKey('document_access_request.id'))
+
+    @property
+    def json(self):
+        """Return a dict of this object, with keys in JSON format."""
+        document = {
+            'id': self.id,
+            'documentType': self.document_type,
+            'documentKey': self.document_key,
+            'fileName': self.file_name or '',
```

Comment:
Is this better than setting it to _None_?

Reply:
No :)